config/prod/minidream-R-env-test-instance-ec2.yaml

```diff
@@ -0,0 +1,20 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v4.yaml
+stack_name: minidream-R-env-test-instance-ec2 
+parameters:
+  # The Sage deparment for this resource
+  Department: "CompOnc"
+  # The Sage project this resource will be used for
+  Project: "CSBC"
+  # The resource owner
+  OwnerEmail: "milen.nikolov@sagebase.org"
+  # EC2 instance type (available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.nano"
+   # Integration with our jumpcloud directory service (do not change)
+  JcConnectKey: !ssm /infra/JcConnectKey
+  JcServiceApiKey: !ssm /infra/JcServiceApiKey
+  JcSystemsGroupId: !ssm /infra/JcSystemsGroupId
+hooks:
```

Comment:
sorry, didn't notice this on the 1st review.  this hooks section should be
```
hooks:
  after_create:
    - !ec2_notify {{stack_group_config.aws_account_name}} {{stack_group_config.aws_account_email}}

```

Reply:
Updated hooks config.